alerting/src/main/kotlin/org/opensearch/alerting/model/DocumentLevelTrigger.kt

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.alerting.model
+
+import org.opensearch.alerting.model.Trigger.Companion.ACTIONS_FIELD
+import org.opensearch.alerting.model.Trigger.Companion.ID_FIELD
+import org.opensearch.alerting.model.Trigger.Companion.NAME_FIELD
+import org.opensearch.alerting.model.Trigger.Companion.SEVERITY_FIELD
+import org.opensearch.alerting.model.action.Action
+import org.opensearch.common.CheckedFunction
+import org.opensearch.common.ParseField
+import org.opensearch.common.UUIDs
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import org.opensearch.common.xcontent.NamedXContentRegistry
+import org.opensearch.common.xcontent.ToXContent
+import org.opensearch.common.xcontent.XContentBuilder
+import org.opensearch.common.xcontent.XContentParser
+import org.opensearch.common.xcontent.XContentParser.Token
+import org.opensearch.common.xcontent.XContentParserUtils
+import org.opensearch.common.xcontent.XContentParserUtils.ensureExpectedToken
+import org.opensearch.script.Script
+import java.io.IOException
+
+/**
+ * A single-alert Trigger that uses Painless scripts which execute on the response of the Monitor input query to define
+ * alerting conditions.
+ */
+data class DocumentLevelTrigger(
```

Comment:
Please make this the same as [this](https://github.com/opensearch-project/alerting/blob/2ea6420e3670d5cfbde8eb115b048ba6fe02267c/alerting/src/main/kotlin/org/opensearch/alerting/model/DocumentLevelTrigger.kt)